packages/docs/operations-manual/validator-explorer.md

```diff
@@ -0,0 +1,83 @@
+# Validator Explorer
+You can interact with the Validator Explorer that allows you to have a complete view of how the different validators are performing. This is one resource voters may use to find validator groups to vote for. The Validator Explorer tool is available in the following address:
+https://dev.celo.org/validators/explore/
+
+All of the existing validators and groups in the Celo network are included in this view. The main view of the tool is organized showing the groups, if you click on any of the group names the affiliates of that group will be displayed. You have the option to sort the contents of the table clicking in the header of each column. 
+
+If you are looking for how your validator is performing, you should be able to find the group to which your validator is affiliated, clicking on the group name will show your validator and the rest of affiliates of the same group.  
+
+A critical element of this explorer is the Validator Group name, which can help voters recognize organizations or active community members. This name is fetched from the Validator Group’s associated `metadata`, which you can read more about [here](https://github.com/celo-org/celo-monorepo/blob/master/packages/docs/celo-codebase/protocol/identity/metadata.md). In order to prevent name impersonation, a group can register a domain claim within their metadata. This is done by adding a [TXT record](https://en.wikipedia.org/wiki/TXT_record) to their domain which includes a signature of their domain claim signed by their associated account. This claim is then verified by the validator explorer. Individual users may also verify a claim using `celocli account:get-metdata` or by manually checking for this `TXT Record` on a group’s purported identity domain. 
```

Comment:
I thought the name is just straight from the `Validators` smart contract?

Reply:
@Pedro-vk @aaitor

Reply:
Yup my bad, the name is actually on the `Account` contract, forgot about this until halfway through the doc and must have missed this part